CONTRIBUTING.md

```diff
@@ -47,47 +47,50 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) was initially forked from [dvc-checkpoints-mnist], and
+is provided as a lightweight, convenient testbed to try the extension out with.
+It is not an exhaustive showcase of DVC's features, so testers are encouraged to
+try other DVC repositories -- especially real-world cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Go to the `demo/` directory of
+  this repo (`cd demo`) and run `yarn install` to set up the project.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Go back to the root of this repo (`cd ..`) and run `yarn setup:venv` to set up
+  a Python virtual environment in the demo project (in `demo/.env`).
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+  > Go to `demo/` if you want to use DVC from command line (terminal).
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- Pull the project data with the extension (**DVC panel** in the _Source Code_
+  view) or with `.env/bin/dvc pull` from command line.
 
-- Once the environment has been setup you should navigate into the demo folder
-  activate the virtual environment and run `dvc pull`.
+- In order to run experiments in the demo project (from the **DVC View**), the
+  virtual env should be loaded through the Microsoft [Python extension] (please
+  install it), having this in `.vscode/settings.json`:
 
-- In order to run experiments in our demo project we require the Python virtual
-  environment to be loaded.
+  ```json
+  "python.defaultInterpreterPath": "demo/.env/bin/dvc"
```

Comment:
BTW why don't we just include this in the repo's .vscode/settings.json file, which is committed to Git?

Reply:
this is not needed if you open `demo` directly